src/components/ViewRoutingSlip/LinkedRoutingSlipDetails.vue

```diff
@@ -28,6 +27,15 @@ export default class LinkedRoutingSlipDetails extends Vue {
 
   public colors = commonUtil.statusListColor
   public formatDisplayDate = commonUtil.formatDisplayDate
+  get formatSiNumber () {
+    let siNumber = ''
+    if (this.siNumber && this.siNumber !== '') {
+      // for padding zero for single digits to match design
+      siNumber = this.siNumber?.toString().padStart(2, '0')
+      siNumber = `${siNumber}.`
```

Comment:
cuirous why returning '#{siNumber}' ; not siNumber alone?

Reply:
As per the design we need add a dot(.) after , adding . and return if number passed. for child details we are not showing number

Reply:
i missed the . 
lol , i thought its some new formatting...
return siNumber+'.' ;isnt that simple..